From dedb1e90daebb4bc3e6d88aa3173464422eb1751 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 29 Nov 2017 16:53:20 +0100
Subject: GCC build fix: access from global function to private methods in
 CJS_PublicMethods.

Build fails on GCC because JSGlobalFunc (a global function) is calling private
methods of CJS_PublicMethods.

Bug: None.

diff --git a/fxjs/cjs_publicmethods.h b/fxjs/cjs_publicmethods.h
index 4088c42f1..87269969f 100644
--- a/fxjs/cjs_publicmethods.h
+++ b/fxjs/cjs_publicmethods.h
@@ -26,7 +26,6 @@ class CJS_PublicMethods : public CJS_Object {
   static WideString MakeFormatDate(double dDate, const WideString& format);
   static bool IsNumber(const WideString& str);
 
- private:
   static CJS_Return AFNumber_Format(
       CJS_Runtime* pRuntime,
       const std::vector<v8::Local<v8::Value>>& params);
-- 
2.14.1

